package.json

```diff
@@ -25,5 +25,8 @@
     "webpack": "^4.42.0",
     "webpack-cli": "^3.3.11",
     "webpack-dev-server": "^3.10.3"
+  },
+  "peerDependencies": {
+    "openmct": "nasa/openmct#release/2.0.0"
```

Comment:
I think that referencing a peer dependency on Open MCT is in principle a good idea. We should reference an npm version of Open MCT though, not a GitHub branch/tag. This is for two reasons: 
1. Peer dependencies will pull down their host library if it's absent, and we _never_ want developers building Open MCT from source just to include it in their project, which is what will happen if we reference a GitHub branch or tag.
2. This doesn't include an explicit Open MCT version identifier that npm can use to figure out whether a build configuration already includes a compatible version of Open MCT.

I realize that's a bit of a catch-22, because the npm version of Open MCT doesn't yet exist when this `package.json` is updated for release _testing_. So we will need to update the peer dependency specifier right before we tag the `openmct-yamcs` release so that it points to the correct npm dependency.